src/components/user/applyComponents/phoneComponent/PhoneComponent.tsx

```
@@ -1,4 +1,8 @@
 import * as S from './PhoneComponent.styled';
+<<<<<<< HEAD
+=======
+import { ApplySchemeType } from '../../../../pages/user/apply/Apply';
+>>>>>>> 4069f4a (chore: 디렉토리 변경에 따른  경로수정)
 import { Control, FieldErrors } from 'react-hook-form';
 import PhoneInput from './phoneComponentInput/PhoneComponentInput';
 import { ApplySchemeType } from '../../../models/joinProject';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'ApplySchemeType'. Consider to delete it or rename it.

'ApplySchemeType' is defined here:


(lint/suspicious/noRedeclare)